include/oneapi/dpl/pstl/hetero/dpcpp/unseq_backend_sycl.h

```diff
@@ -80,6 +80,36 @@ inline constexpr _Tp __known_identity =
     __known_identity_for_plus<_BinaryOp, _Tp>::value; //for plus only
 #endif //_ONEDPL_LIBSYCL_VERSION >= 50200
 
+// a way to get value_type from both accessors and USM that is needed for transform_init
+template <typename _Unknown>
+struct __accessor_traits_impl
+{
+};
+
+template <typename _T, int _Dim, sycl::access::mode _AccMode, __dpl_sycl::__target _AccTarget,
+          sycl::access::placeholder _Placeholder>
+struct __accessor_traits_impl<sycl::accessor<_T, _Dim, _AccMode, _AccTarget, _Placeholder>>
```

Comment:
I think that this old implementation for ```sycl::accessor``` still required in our code because we still have usage of ```sycl::accessor``` in our code and it's OK.

Reply:
This comment is not actual after new proposed variant of compile error fix.